### PR TITLE
Add catalyst

### PR DESCRIPTION
**Pull request by manjanac**
Hey there! So you want to contribute to a Firebase SDK?
Before you file this pull request, please read these guidelines:

### Discussion

  * Read the contribution guidelines (CONTRIBUTING.md).
  * If this has been discussed in an issue, make sure to link to the issue here.
    If not, go file an issue about this **before creating a pull request** to discuss.

### Testing

  * Make sure all existing tests in the repository pass after your change.
  * If you fixed a bug or added a feature, add a new test to cover your code.

### API Changes

  * At this time we cannot accept changes that affect the public API.  If you'd like to help
    us make Firebase APIs better, please propose your change in a feature request so that we
    can discuss it together.

### CODE REVIEW

File: .github/workflows/mlmodeldownloader.yml [hunk 1]
```diff
@@ -52,8 +52,7 @@ jobs:
     runs-on: macOS-latest
```
Comment (paulb777) [inline .github/workflows/mlmodeldownloader.yml:52]: Temporarily comment line 51 to test the `spm-cron` job before tonight.

File: .github/workflows/mlmodeldownloader.yml [hunk 2]
```diff
@@ -65,9 +64,7 @@ jobs:
 
   catalyst:
     # Don't run on private repo unless it is a PR.
-    # TODO: manjanac@ Uncomment line below to re-enable catalyst.
-    # if: github.repository != 'FirebasePrivate/firebase-ios-sdk' || github.event_name == 'pull_request'
-    if: false
+    if: github.repository != 'FirebasePrivate/firebase-ios-sdk' || github.event_name == 'pull_request'
     runs-on: macOS-latest
     steps:
     - uses: actions/checkout@v2
```
Comment (paulb777) [inline .github/workflows/mlmodeldownloader.yml:70]: Change the `build` on line 74 to `test` to also run tests

File: .github/workflows/mlmodeldownloader.yml (hunk 1 above)
Comment (manjanac) [reply .github/workflows/mlmodeldownloader.yml:52]: Done!

File: .github/workflows/mlmodeldownloader.yml (hunk 2 above)
Comment (manjanac) [reply .github/workflows/mlmodeldownloader.yml:70]: I made this change, but my integration tests are failing. Am I missing the GoogleService-Info.plist under `catalyst`?

File: .github/workflows/mlmodeldownloader.yml (hunk 1 above)
Comment (paulb777) [reply .github/workflows/mlmodeldownloader.yml:52]: LGTM!

File: .github/workflows/mlmodeldownloader.yml (hunk 2 above)
Comment (paulb777) [reply .github/workflows/mlmodeldownloader.yml:70]: Usually a `GoogleService-Info.plist` isn't needed for unit tests. It looks like the scheme needs to be specified - like https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/core.yml#L70

Comment (manjanac) [reply .github/workflows/mlmodeldownloader.yml:70]: Ah perfect, thanks!

**Review by paulb777 — APPROVED** [review]
LGTM! after uncommenting cron

**Pull request merged**